maro/simulator/scenarios/finance/account.py

```diff
@@ -0,0 +1,39 @@
+from collections import OrderedDict
+
+from maro.backends.frame import NodeAttribute, NodeBase, node
+from maro.simulator.scenarios.finance.common import Order, TradeResult, two_decimal_price
```

Comment:
```
from maro.backends.frame import NodeAttribute, NodeBase, node
[one blank line]
from .common import ...
```

similar to other files in folder finance/

Reply:
done